### ISSUE TITLE
Missing poldrack-categorize plugin when running https://expfactory-experiments.github.io/stroop/

### ISSUE THREAD

**Issue by earcanal** [expfactory-experiments/stroop#2]
I get the following error when running the Stroop demo:

    Failed to load resource: the server responded with a status of 404 ()
    120jspsych.js:273 No plugin loaded for trials of type "poldrack-categorize"
    (anonymous) @ jspsych.js:273 /stroop/%3Cdiv%20class%20=%20centerbox%3E%3Cdiv%20class%20=%20fixation%3E+%3C/div%3E%3C/div%3E Failed to load resource: the server responded with a status of 404 ()
    jspsych.js:581 Uncaught TypeError: Cannot read property 'trial' of undefined
        at doTrial (jspsych.js:581)
        at next_trial (jspsych.js:167)

I've submitted a PR for this (untested) which might fix it.

**Comment by vsoch** [comment]
Any reason you didn't test? It's all static html so you should be able to start a local webserver in the folder, or enable GitHub pages for master in your fork and test there.

**Comment by earcanal** [comment]
Sorry, was on a Chromebook at the time.  Previous PR was rubbish.  New one passes my test (doesn't even need a web server as you can File->Open `index.html`:).

Should CI pick up this sort of issue?

**Comment by vsoch** [comment]
Fixed with https://github.com/expfactory-experiments/stroop/pull/3

@earcanal this is definitely a big issue when it comes to experiment development - how do we test them? I created a "testing robot" for experiments that conformed to jspsych, and likely I could do a version 2.0 for the new expfactory (and I'm a better programming now too). It still would be an issue for other experiments that don't use jspych, but a good start!
